### ISSUE TITLE
Package.activateConfig is deprecated.

### ISSUE THREAD

**Issue by rabugento** [AtomLinter/linter-php#18]
Hi, please fix this error. Thanks.

Use a config schema instead. See the configuration section
of https://atom.io/docs/latest/hacking-atom-package-word-count and
https://atom.io/docs/api/latest/Config for more details
```
Package.activateConfig (/usr/share/atom/resources/app.asar/src/package.js:267:11)
Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:238:14)
<unknown> (/usr/share/atom/resources/app.asar/src/package.js:223:30)
Package.measure (/usr/share/atom/resources/app.asar/src/package.js:167:15)
```

**Comment by taeram** [comment]
I'm seeing this same issue, as reported by the Deprecation Cop in Atom 0.196.0, but with slightly different line numbers:

Use a config schema instead. See the configuration section
of https://atom.io/docs/latest/hacking-atom-package-word-count and
https://atom.io/docs/api/latest/Config for more details
```
Package.activateConfig (C:\Atom\resources\app.asar\src\package.js:247:11)
Package.activateNow (C:\Atom\resources\app.asar\src\package.js:218:14)
<unknown> (C:\Atom\resources\app.asar\src\package.js:203:30)
Package.measure (C:\Atom\resources\app.asar\src\package.js:147:15)
```

**Comment by kevinsawicki** [comment]
Fixed by #29  and published in `0.0.13`
